components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -80,15 +80,28 @@ Views.loginOpened...
 There are different metrics to choose from, depending on what you want to achieve:
 
 * [Events](#Events): Records events e.g. individual occurences of user actions, say every time a view was open and from where.
-* [Booleans](#Booleans): Records a single truth value, for example "is a11y enabled?"
+* [Booleans](#Booleans): Records a single truth value, for example "is a11y enabled?".
+* [Labeled booleans](#Labeled-booleans): Records multiple truth values, for example which URI schemes were used or not - `"http"`, `"https"`, `"about"`.
 * [Strings](#Strings): Records a single Unicode string value, for example the name of the OS.
-* [String Lists](#String-lists): Records a list of Unicode string values, for example the list of enabled search engines.
+* [Labeled strings](#Labeled-strings): Records multiple Unicode string values. **TODO: what's the use-case here? Should we block these for now?**
```

Comment:
Labeled strings seem to overlap quite a bit with string lists.  I'd be okay with keeping them out until a strong use-case comes up.

Reply:
Let's file a bug about this, please.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1557349

Reply:
Per the latest comments, this is in use by sync.
We should keep it for now and defer the discussion.
https://bugzilla.mozilla.org/show_bug.cgi?id=1557349#c2